cristin-project/src/test/java/no/unit/nva/cristin/projects/FetchCristinProjectsTest.java

```diff
@@ -113,7 +115,8 @@ void setUp() {
     @ArgumentsSource(TestPairProvider.class)
     void handlerReturnsExpectedBodyWhenRequestInputIsValid(String expected) throws IOException {
         String actual = sendDefaultQuery().getBody();
-        assertEquals(OBJECT_MAPPER.readTree(expected), OBJECT_MAPPER.readTree(actual));
+        assertEquals(OBJECT_MAPPER.readValue(expected,GatewayResponse.class),
+                OBJECT_MAPPER.readValue(actual, GatewayResponse.class));
```

Comment:
Nice

Reply:
It fails, hopefully..?

Reply:
It doesn't as it OBJECTMAPPER always returns null...

Reply:
So its a false positive then?

Reply:
Fixed